Tree_Inorder_Traversal/Tree_Inorder_Traversal.js

```diff
@@ -0,0 +1,34 @@
+// Creates a Tree Node with input value
```

Comment:
brief problem description on top.

Reply:
@singh-shreya6 Did the asked changes. Please review it and do let me know if any changes are required.